plugins/woocommerce/src/Blocks/BlockTypes/ProductCollection.php

```diff
@@ -304,14 +304,16 @@ private function enable_client_side_navigation( $block_content ) {
 			$p->set_attribute(
 				'data-wc-context',
 				wp_json_encode(
-					array(
-						...$current_context,
```

Comment:
Wasn't this introduced in 7.4?

Reply:
While the splat operator _was_ introduced in 7.4, [you couldn't use it on associative arrays until PHP 8.1](https://www.php.net/releases/8.1/en.php#array_unpacking_support_for_string_keyed_arrays)!

Reply:
Thanks for catching this @ObliviousHarmony 💜